lighthouse-core/report/v2/report-renderer.js

```diff
@@ -64,15 +71,31 @@ window.ReportRenderer = class ReportRenderer {
   }
 
   /**
-   * @param {!ReportJSON} report
-   * @return {!Element}
+   * Sets the text content of an element, safely. If the element does not exist,
+   * results in a noop.
+   * @param {Element} element
+   * @param {!string} text
+   * @return {Element}
    */
-  renderReport(report) {
-    try {
-      return this._renderReport(report);
-    } catch (e) {
-      return this._renderException(e);
+  static setText(element, text) {
+    if (element) {
```

Comment:
if we're worried about these, should they just throw? Seems like it's definitely an error since we're creating the templates *and* the selectors looking within them